### PR TITLE
Sample Historical Data JSON Requests

### PR DESCRIPTION
**Pull request by aceofsteel**
Add sample JSON requests to repo

### CODE REVIEW

**Comment by ericvw** [comment]
These look great!  Two things:
1. Can you indent the JSON by either 2 or 4 spaces?
2. Once you are done, please squash all these commits into one single commit.

Thanks!

**Comment by aceofsteel** [comment]
Updated Requests and made into single commit.

File: examples/json/HistoricalData/AverageUSGasPrices.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+"securities": ["3AGSREG Index"],
```
Comment (ericvw) [inline examples/json/HistoricalData/AverageUSGasPrices.json]: Let's indent this file as well

**Comment by ericvw** [comment]
For all `.sh` files, let's make sure to wrap all `$VAR` expansions in double quotes (i.e., `"`) so they don't undergo splitting based on `IFS`.

**Comment by ericlu88** [comment]
LGTM

**Comment by ericvw** [comment]
I agree with @yupinghu, let's order the JSON by:
1. Securities
2. Fields
3. Start date
4. End date
5. ... anything else

**Comment by aceofsteel** [comment]
Quoted args and vars in shell script
Updated ordering of json

**Comment by ericvw** [comment]
lgtm.  @ericlu88 or @yupinghu - take another glance over and if it looks good by either you, please feel free to merge.

**Pull request merged**